PCbuild/fix_lineendings.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+#
+# Fixes line endings of batch files to ensure they remain LF.
```

Comment:
I needed the script to fix up the problems when hggit did the wrong thing. Maybe I just need to switch to git completely (though then I'm *never* going to backport anything, because it takes 10x as long)